pages/am/3.x/user-guide/identity-provider/userguide-enterprise-identity-provider-http.adoc

```diff
@@ -0,0 +1,239 @@
+= HTTP (web service)
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_userguide_enterprise_identity_provider_http.html
+:page-folder: am/user-guide
+:page-layout: am
+
+Gravitee.io AM allows you to authenticate and manage users via remote API calls.
+
+WARNING: For the Identity Provider to work, the user's payload must at least contain the following claims : _sub_.
+To have a more information about your user you can rely on the link:https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[Standard Claims] of the OpenID Connect protocol.
+To achieve this you can use the User Mapper feature of Gravitee.io AM.
+
+image::am/current/graviteeio-am-userguide-http-idp-mapping.png[]
+
+== Create a HTTP identity provider
+
+Identity providers are created for each security domain, to create an identity provider, follow these steps:
+
+. Go to http://GRAVITEEIO-AM-UI-HOST
+. From the homepage, go to your security domain's `Settings` page, go to `Providers` section
+. From the security domain's providers page, click `(+)` button
+. Choose the `HTTP` identity provider and press `NEXT`
+. Give your identity provider a name
+. Configure the settings as requested
+. Press `Create`
+
+=== Configuration
+
+The HTTP Identity Provider is used to invoke an HTTP(S) URL and put the response content into one or more variables of the execution context.
+
+The result of the authentication is put in a variable named _authenticationResponse_ and the User Management operations are stored in the _usersResponse_ variable.
+
+These variables can be used to check if the result API calls have failed or not. To configure your Identity Provider, you can find after all the data context and options to suit your needs.
```

Comment:
whether the API calls have failed. 
The sections below list the data context and options you can configure for your identity provider.